packages/components/src/product-icon/index.jsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import paths from './paths';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+function ProductIcon( { className, slug } ) {
+	return (
+		<img
+			src={ paths[ slug ] }
+			className={ classNames(
+				'product-icon',
+				`product-icon__${ slug }`,
+				`is-${ slug }`,
```

Comment:
The `product-icon__${ slug }` and `is-${ slug }` seem to be redundant -- they serve the same need (customize by specific icon) and we need only one.

Can we clean this up in the process of moving the component to `@automattic/components`?

Reply:
Sounds perfect! 

Originally, in the `PlanIcon` component, the modifier was in a form of `product-icon__${ slug }`. I left for backwards compatibility. I haven't actually checked if it's used at all, though.

I will go through the codebase and replace any occurrences with a `is-${ slug }` (which seems to be a more up-to-date pattern in terms of our coding standards).